### PR TITLE
Basic localization

### PR DESCRIPTION
**Pull request by miaerbus**
Hello, it's me again.

This PR fixes @lukarenko points from Slack:
- Dodatki link na spletno stran (lahko kar link na ime institucije npr. ZD Ljubljana)
- O projektu: bom napisal nekaj o ideji in avtorjih. Najbolje v .md, kot na glavni strani?
- Prijavi napako: jaz bi naredil podpora-zdravniki@sledilnik.org in bi enostavno naredil mailto: z subjectom. Za začetek je to dovolj, potem bomo pa videli koliko je tega in kako se lotiti.
- Priprava za lokalizacije - vsaj angleščina bi prav prišla za začetek, ker marsikateri tujec ne najde zdravnika.

Sorry for a huge commit. I know we talked about not using scss. I used it again for the About page, because it looks quite challenging to me to style it with MUI. Perhaps we can keep it until someone decides to try to style it with MUI instead?
I had some problems using nested parametrized routes with React Router 6. I hard coded the routes for now, will look into that later again.
Also added 'webpack-bundle-analyzer', so we can check for the bundle size. At first glance, MUI components add quite a lot to the project size.

### CODE REVIEW

**Comment by jalezi** [comment]
screenshot

Something is broken.


<details><summary>Browser metadata</summary>





```
Path:      /sl/about
Browser:   Chrome 96.0.4664.45 on Windows 10
Viewport:  831 x 765 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-9--prosti-zdravniki.netlify.app%2Fsl%2Fabout&os=Windows&os_version=10&browser=Chrome&browser_version=96.0&resolution=831x765&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-9--prosti-zdravniki.netlify.app/sl/about) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=619e7b9a1acef000082193a0&commentID=619e8fbda89ea29f8cbbb567&resolution=resolved)

**Comment by jalezi** [comment]
@miaerbus I will try to convert `scss`. It shouldn't be too difficult.

**Comment by miaerbus** [comment]
Oops, will fix it. And also move the language switcher into hamburger. It would be great if you could convert the scss. Thank you!

**Comment by jalezi** [comment]
> It would be great if you could convert the scss. Thank you!

Done! https://gist.github.com/jalezi/c9ca0a1eff06905c6f296e942f76f8fb

File: src/content/sl/about.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## Pogoji uporabe
+
+Uporaba podatkov, grafov in sodelovanje so zaželjeni: podatki so zbrani iz virov v javni domeni in jih lahko prosto uporabljate, urejate, predelujete ali vključujete v vse netržne vsebine ob navedbi vira – zdravniki.sledilnik.org. Če ni določeno drugače, velja za vso vsebino na tej strani licenca Creative Commons: Priznanje avtorstva-Deljenje pod enakimi pogoji 3.0.
+
+Za druge oblike sodelovanja nas kontaktirajte na podpora-zdravniki@sledilnik.org.
+
+## Varstvo osebnih podatkov in piškotki
+
+VPodajamo informacije o obdelavah osebnih podatkov po 13. in 14. členu oz. 30. členu Splošne uredbe o varstvu podatkov ter prostemu pretoku podatkov (GDPR) ter o rabi spletnih piškotkov po 157. členu Zakona o elektronskih komunikacijah (ZEKom-1).
```
Comment (jalezi) [inline src/content/sl/about.md:29]: typo?

Comment (miaerbus) [reply src/content/sl/about.md]: Fixed, thank you!

**Comment by miaerbus** [comment]
> Done! https://gist.github.com/jalezi/c9ca0a1eff06905c6f296e942f76f8fb
Awesome! Do you want to add it directly to this PR?

**Comment by jalezi** [comment]
Sure, If you have time. I just wonder if we should leave `.scss` file for reference. I've included only necessary styling.

I'm also working on lang-selector style. Below is what I have at moment.

Screenshot 2021-11-25 15 16 07

**Comment by jalezi** [comment]
Here is what I have so far for `<Autocomplete/>` if you want to play around. No pressure. We can do it later.

[custom-autocomplete-popper.zip](https://github.com/jalezi/find-available-doctor/files/7603774/custom-autocomplete-popper.zip)

**Comment by jalezi** [comment]
@miaerbus firstly, I am sorry I haven't pushed to your PR.

I am not sure what is the plan with doctor page. Probably I mislead you. Maybe we don't need special routes and open it as a modal. I will ask on Slack later today in case decision was not already made.

At the moment I am trying to solve conflicts between our branches. Actually I did and I will try to push it to your branch later today.

Anyway, whatever we decide, we can solve map differently.
I used to have doctor's map inside the `DoctorCard`. You can check it out in 0c484e7ef23fc676d6b14ba8e44dd84a0e245a59 and ed504fd096d98d084a3f84b7f211d7dcfe58c27b and we can talk about it.

**Comment by lukarenko** [comment]
@jalezi subpage makes sense for SEO, this is why I like the route that @miaerbus did
I think we should just merge the branches and release :)

**Comment by jalezi** [comment]
@lukarenko no problem, I was not sure what we want.

Unfortunately it's not ready for release :) Will be. Soon. A few minor bugs. Some cleanup.

Branches are merged in #11.

**Comment by lukarenko** [comment]
@jalezi I think more about soft-launch release that we would share with ZZZS, Zdravni[ka zbornica, doctors to collect some preview feedback.

**Comment by stefanb** [comment]
@miaerbus please check if any part of this PR was not included in #11.

**Comment by lukarenko** [comment]
Yes, this was included and is obsolete. Closing.

**Pull request closed without merging**